swagger-java-quay-client/pom.xml

```diff
@@ -33,14 +34,19 @@
             <plugin>
                 <groupId>io.swagger</groupId>
                 <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>2.2.2</version>
```

Comment:
There's something strange going on here. 
This generates the swagger classes as a client for talking to quay.io